src/main/java/io/github/nucleuspowered/nucleus/commands/kit/KitSetCooldownCommand.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * This file is part of Nucleus, licensed under the MIT License (MIT). See the LICENSE.txt file
+ * at the root of this project for more details.
+ */
+package io.github.nucleuspowered.nucleus.commands.kit;
+
+import com.google.inject.Inject;
+import io.github.nucleuspowered.nucleus.Util;
+import io.github.nucleuspowered.nucleus.config.KitsConfig;
+import io.github.nucleuspowered.nucleus.internal.CommandBase;
+import io.github.nucleuspowered.nucleus.internal.annotations.Permissions;
+import io.github.nucleuspowered.nucleus.internal.annotations.RegisterCommand;
+import io.github.nucleuspowered.nucleus.internal.permissions.SuggestedLevel;
+import org.spongepowered.api.command.CommandResult;
+import org.spongepowered.api.command.args.CommandContext;
+import org.spongepowered.api.command.args.GenericArguments;
+import org.spongepowered.api.command.spec.CommandSpec;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.text.Text;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Sets kit items.
+ *
+ * Command Usage: /kit set Permission: nucleus.kit.set.base
+ */
+@Permissions(root = "kit", suggestedLevel = SuggestedLevel.ADMIN)
+@RegisterCommand(value = {"setcooldown"}, subcommandOf = KitCommand.class)
+public class KitSetCooldownCommand extends CommandBase<Player> {
+
+    @Inject private KitsConfig kitConfig;
+
+    private final String name = "name";
+    private final String timeUnit = "time unit";
+    private final String cooldown = "cooldown";
+
+    @Override
+    public CommandSpec createSpec() {
+        return CommandSpec.builder().executor(this).description(Text.of("Sets kit cooldown."))
+                .arguments(GenericArguments.seq(GenericArguments.onlyOne(GenericArguments.string(Text.of(name))),
+                        GenericArguments.onlyOne(GenericArguments.string(Text.of(timeUnit))),
+                        GenericArguments.onlyOne(GenericArguments.integer(Text.of(cooldown)))))
```

Comment:
I think we should be considering using my duration class, rather than this. It's the same as the `1d1h1m1s` format, though we can extend it. That way, we're consistent with the rest of the plugin.